scwconfig/README.md

```diff
@@ -0,0 +1,27 @@
+# Scaleway config
+
+## TL;DR
+
+Recommended config file:
+
+```yaml
+# get your access and secret keys on https://console.scaleway.com/account/credentials
+access_key: SCWXXXXXXXXXXXXXXXXX
+secret_key: xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+default_organization_id: xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx
+default_region: fr-par
+default_zone: fr-par-1
+```
+
```

Comment:
Can you add a list of the priority order, for ex: (a low number have a high priority)
1. varenv
2. deprecated var env
3. config file (v2)
4. deprecated config file (v1) If this file is found an automatic migration (v1 => v2) is made.